src/test/java/uk/gov/hmcts/reform/hmc/helper/CaseHearingRequestMapperTest.java

```diff
@@ -44,7 +45,6 @@ void modelToEntity() {
         assertEquals("CMLC123", entity.getOwningLocationId());
         assertEquals(Boolean.FALSE, entity.getCaseRestrictedFlag());
         assertEquals(Boolean.FALSE, entity.getCaseRestrictedFlag());
-        assertEquals(1, entity.getVersionNumber());
```

Comment:
Are we ok losing this assert?

Reply:
Code has been refactored to use same methods for PUT and POST methods. Version number is validated in IT tests.